src/TodoList.js

```diff
@@ -1,5 +1,28 @@
 class TodoList {
+  constructor() {
+    this.todos = []; // After the constructor is ran, this is the state of the instance.
+  }
 
+  create(text) {
+    const todoToCreate = {
+      id: this.todos.length + 1,
+      text: text,
+      status: "incomplete",
+    };
+
+    this.todos.push(todoToCreate);
+
+    return todoToCreate;
+  }
+
+  allToDoItems() {
+    return this.todos;
+  }
```

Comment:
What do you think stopped you from moving forward after completing the allToDoItems requirement?